src/components/views/avatars/DecoratedRoomAvatar.tsx

```diff
@@ -35,7 +35,11 @@ interface IProps {
     room: Room;
     size: string;
     displayBadge?: boolean;
-    forceCount?: boolean;
+    /**
+     * If true, show nothing if the notification would only cause a dot to be shown rather than
+     * a badge. That is: only display badges and not dots. Default: false.
+     */
+    hideIfDot?: boolean;
```

Comment:
So the the reason I didn't go with this is that it would be something like, 'showIfDot', ie. whether to show the component if it would only be a dot. Having it as 'hide' felt more apparent that it was talking about hiding the component itself, rather than some part of the component? It is still, admittedly, a slightly weird option.